src/doc_scan_service/session/retrieve/resource.container.js

```diff
@@ -0,0 +1,64 @@
+'use strict';
+
+const DocumentResourceResponse = require('./document.resource.response');
+const ZoomLivenessResourceResponse = require('./zoom.liveness.resource.response');
+const LivenessResourceResponse = require('./liveness.resource.response');
+const DocScanConstants = require('../../doc.scan.constants');
+const Validation = require('../../../yoti_common/validation');
+
+class ResourceContainer {
+  constructor(resources) {
+    if (resources.id_documents) {
+      Validation.isArray(resources.id_documents, 'id_documents');
+      this.idDocuments = resources
+        .id_documents
+        .map(resource => new DocumentResourceResponse(resource));
```

Comment:
This should be `new IdDocumentResourceResponse(resource)`

Reply:
Renamed in d896ddd